samples/net/sockets/echo_server/boards/intel_socfpga_agilex5_socdk.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2023 Intel Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* The overlay file should be used to enable any
+ * dts nodes required by this application for this
+ * board.
+ */
+
+&xgmac0 {
+	status = "okay";
+	local-mac-address = [06 00 00 00 00 02];
```

Comment:
same question here for the mac address

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/65869#discussion_r1416213099